data-serving/data-service/api/openapi.yaml

```diff
@@ -197,6 +197,16 @@ paths:
           $ref: "#/components/responses/422"
         "500":
           $ref: "#/components/responses/500"
+  /cases/download:
+    get:
+      summary: Streams a CSV attachment of all cases
+      tags: [Case]
+      operationId: downloadCases
+      responses:
+        "200":
+          $ref: "#/components/responses/200"
```

Comment:
can we describe a bit more what the response body is like here?

Reply:
Done.